README.md

```diff
@@ -3,7 +3,7 @@
 
 An OSBAPI-compatible broker written in golang.
 
-Configure the broker, by setting `username` and `password` in `config.json`.
+Configure the broker, by setting `username` and `password` in `config.json`. Update the `deployer_path` in `config.json` to point to the location of `pusher.js` in `$PWD/pkg/utils/pusher.js`.
```

Comment:
don't think it needs the $PWD, you can make it link directly to the file. 
`[pusher.js](pkg/utils/pusher.js)`